lua/oil/actions.lua

```diff
@@ -337,6 +337,55 @@ M.toggle_trash = {
   end,
 }
 
+M.send_to_qflist = {
+  desc = "Sends files in the current oil directory to the quickfix list, replacing the previous entries.",
+  callback = function()
+    util.send_to_quickfix({
+      directories = false,
+      files = true,
```

Comment:
I don't think that `directories = true` is going to function correctly with the given logic, but if we don't have an immediate use for these parameters (seems like it's always `directories = false` and `files = true`), then I'd err on the side of removing them.